core/blockchain.go

```diff
@@ -342,7 +357,34 @@ func ExecuteBlockEphemerally(
 		}
 	}
 
-	return receipts, stateSyncReceipt, nil
+	execRs := &ExecutionResultFateme{
+		// StateRoot:   root,
+		TxRoot:      types.DeriveSha(includedTxs),
+		ReceiptRoot: types.DeriveSha(receipts),
+		Bloom:       types.CreateBloom(receipts),
+		LogsHash:    rlpHash(ibs.Logs()),
+		Receipts:    receipts,
+		Rejected:    rejectedTxs,
+	}
+
+	return receipts, stateSyncReceipt, execRs, nil
+}
+
+type ExecutionResultFateme struct {
+	// StateRoot   common.Hash    `json:"stateRoot"` I think we don't need this
+	TxRoot      common.Hash    `json:"txRoot"`
+	ReceiptRoot common.Hash    `json:"receiptRoot"`
+	LogsHash    common.Hash    `json:"logsHash"`
+	Bloom       types.Bloom    `json:"logsBloom"        gencodec:"required"`
+	Receipts    types.Receipts `json:"receipts"`
+	Rejected    []*rejectedTx  `json:"rejected,omitempty"`
+}
+
+func rlpHash(x interface{}) (h common.Hash) {
```

Comment:
can we reused hashing.go:rlpHash function? rather than reimplementing this here?

Reply:
the `hashing.go:rlpHas` is not exported